pkg/handler/handler.go

```diff
@@ -274,6 +274,7 @@ func (h handler) provision(w http.ResponseWriter, r *http.Request, params map[st
 		return
 	}
 
+	var username = "default"
```

Comment:
this is a little weird. I would've left it empty and if it is still empty when we call the apb, we don't pass anything to the apb.

Reply:
I can update this...